examples/06-kitchen-sink/src/block/block.js

```diff
@@ -0,0 +1,482 @@
+/**
+ * BLOCK: kitchen-sink
+ *
+ * Registering a basic block with Gutenberg.
+ * Simple block, renders and saves the same content without any interactivity.
+ */
+
+//  Import CSS.
+import "./style.scss";
+import "./editor.scss";
+
+const { __ } = wp.i18n; // Import __() from wp.i18n
+const { registerBlockType } = wp.blocks; // Import registerBlockType() from wp.blocks
+const { Fragment } = wp.element; // Import Fragment from wp.elements
+
+// Register editor components
+const {
+	RichText,
+	PlainText,
+	AlignmentToolbar,
+	BlockControls,
+	MediaUpload,
+	InnerBlocks,
+	PanelColorSettings,
+	InspectorControls,
+	URLInput
+} = wp.editor;
+
+// Import Inspector components
+const {
+	IconButton,
+	Toolbar,
+	Button,
+	PanelBody,
+	PanelRow,
+	ToggleControl,
+	RangeControl,
+	TextControl,
+	TextareaControl,
+	SelectControl
+} = wp.components;
+
+// Declare attributes to save various data in block: https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-attributes/
+const kitchenAttributes = {
+	exampleToggle: {
+		type: "boolean",
+		default: false
+	},
+	exampleRange: {
+		type: "number",
+		default: 18
+	},
+	exampleSelect: {
+		type: "string",
+		default: "option_1"
+	},
+	exampleText: {
+		type: "string"
+	},
+	exampleTextarea: {
+		type: "string"
+	},
+	exampleColor: {
+		type: "string",
+		default: "#FFFFFF"
+	},
+	exampleAlignment: {
+		type: "string"
+	},
+	exampleRichH2: {
+		type: "string",
+		source: "children",
+		selector: "h2"
+	},
+	exampleRichParagraph: {
+		type: "string",
+		source: "html",
+		selector: "p"
+	},
+	exampleRichList: {
+		type: "string",
+		selector: "ul"
+	},
+	examplePlainText: {
+		type: "string"
+	},
+	exampleLink: {
+		type: "string"
+	},
+	exampleImageAlt: {
+		type: "string"
+	},
+	exampleImageURL: {
+		type: "string",
+		source: "attribute",
+		attribute: "src",
+		selector: "img"
+	},
+	exampleWidth: {
+		type: "string",
+		default: ""
+	}
+};
+
+/**
+ * Register: aa Gutenberg Block.
```

Comment:
`aa Gutenberg Block.`   Maybe rename that `aa` here. Docs should be up to date. :)